lib/phoenix/code_reloader.ex

```diff
@@ -187,4 +187,29 @@ defmodule Phoenix.CodeReloader do
 
   defp method(%Plug.Conn{method: method}), do:
     method
+
+  defp before_send_inject_reloader(conn) do
```

Comment:
It is OK for this to be here for now but we should seriously consider this to be another plug. :)